include/nbl/core/decl/smart_refctd_ptr.h

```diff
@@ -98,6 +98,11 @@ class smart_refctd_ptr
 		inline I_REFERENCE_COUNTED& operator[](size_t idx) { return ptr[idx]; }
 		inline const I_REFERENCE_COUNTED& operator[](size_t idx) const { return ptr[idx]; }
 
+		[[nodiscard]]
+		bool isValid() const noexcept
+		{
+			return ptr != nullptr;
+		}
```

Comment:
we already define an `opeerator(bool)`

So
```cpp
if (!ptr)
if (bool(ptr))
```
should work.

Please remove

Reply:
Ok